app/ui-react/pom.xml

```diff
@@ -386,10 +390,6 @@
                 <id>yarn ng lint</id>
                 <phase />
               </execution>
-              <execution>
```

Comment:
This is confusing, `yarn ng` is for Angular. Where is the stuff to build the React stuff?

Reply:
it is outside of the `flash` profile i.e. https://github.com/syndesisio/syndesis/blob/2b063db329cbc9a76b0773d0db4a3b62dcf28943/app/ui-react/pom.xml#L94-L162 so if you say that yarn ng is useless I can remove more stuff from that profile and leave just the skip test property to `true` (which is the bit actually disabling the tests).

Reply:
Useless I don't know, surely it's not needed to build the React app. Mind, there are two Angular based projects that we build when building the main app, but that's taken care for by the top-level `yarn build` command. The fact that under the hood there is `ng` involved should not bleed in the POM I think.

I think this is just a leftover from the conf that we used to build the Angular version.

Reply:
I think so as well, the `ui-react/pom.xml` has clearly started as a copy/paste from the angular one. So removed all of those.